graph/src/data/subgraph/mod.rs

```diff
@@ -666,6 +673,59 @@ impl UnresolvedMapping {
     }
 }
 
+#[derive(Clone, PartialEq, Debug)]
+pub struct UnifiedMappingApiVersion(Option<Version>);
+
+impl UnifiedMappingApiVersion {
+    pub fn equal_or_greater_than(&self, other_version: &Version) -> bool {
+        assert!(
+            other_version >= &API_VERSION_0_0_5,
+            "api versions before 0.0.5 should not be used for comparison"
+        );
+        match &self.0 {
+            Some(version) => version >= other_version,
+            None => false,
+        }
+    }
```

Comment:
I was having some broad lifetimes issues when using:

```rust
other_version: &'static Version
``` 

So I removed the `'static` lifetime bound.